.github/workflows/bump-version-and-update-milestone.yml

```diff
@@ -0,0 +1,36 @@
+#############################################################################
+# GitHub Action to bump release version
+#
+#############################################################################
+name: "Bump version"
+on:
+  push:
+    tags:
+    - 'bump-version-to-*'
```

Comment:
I think this action should be not tag based, lets use manual execution with input from user
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

action should be allowed to executed only by maintainers.

Reply:
Changes Done